markdown/docs/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Docs
+subtitle: Here you can find all documentation on how to use nf-core pipelines, and nf-core tools to start processing your data. You will also find information on how to contribute to nf-core pipelines, modules and subworkflows.
+---
+This section contains the following subsections:
+<!-- inline_toc -->
```

Comment:
this renders currently only the `usage` section of the tocs. I would find it more useful with the complete ToC, with nested unordered lists (bonus points if we do it in a responsive two column layout 🤔)

Reply:
Think I got it now :-)